### ISSUE TITLE
Parameter for `prisma init` to supply database provider string to be used in `datasource`

### ISSUE THREAD

**Issue by janpio** [prisma/prisma#2779]
## Problem
Although I always use a MySQL database, I can not create a project with a `provider = 'mysql'`.

## Suggested solution
Similar to https://github.com/prisma/prisma/issues/2577 it would be neat to be able to set (just) the provider of the datasource via a param `--provider`.

## Alternatives
Do it manually.

**Comment by SuryaElavazhagan** [comment]
I tried to fix this here: #7481

**Comment by Jolg42** [comment]
Thanks to @SuryaElavazhagan for the PR!

I did small changes in https://github.com/prisma/prisma/pull/7557

So how this works?
You can now do with latest internal dev version
```
npx prisma init (default to postgresql)
npx prisma init --provider mysql
npx prisma init --provider sqlserver
npx prisma init --provider sqlite
npx prisma init --provider postgresql
npx prisma init --provider mongodb

the provider is lowercased at runtime so passing MySQL as a provider works too.

npx prisma init --provider invalid 
# will return:
Provider "invalid" is invalid or not supported. Try again with "postgresql", "mysql", "sqlserver" or "sqlite".
```

**Comment by janpio** [comment]
Great work @SuryaElavazhagan - thanks so much! Small feature, but sooo convenient when you are a MySQL user usually.

**Comment by Jolg42** [comment]
We noticed an ambiguity with the new --provider argument.

Since both the generator and datasource blocks have a provider field, it's not clear (just from seeing the argument name --provider) to which of them it applies.

We just removed that --provider from the latest dev version so it doesn't get released to decide if we want to do something to release it later.

PR is https://github.com/prisma/prisma/pull/7671
